src/app/app-routing.module.ts

```diff
@@ -3,12 +3,15 @@ import {RouterModule, Routes} from '@angular/router';
 import { UserFormComponent } from './user-form/user-form.component';
 import { EventListComponent } from './event-list/event-list.component'
 import { EventFormComponent } from './event-form/event-form.component';
+import { SearchResultsComponent } from './search-results/search-results.component';
+
 
 const routes: Routes = [
   // {path: 'list-user', redirectTo: '/', pathMatch: 'full'},
   { path: 'events', component: EventListComponent },
   {path: 'addevent', component: EventFormComponent},
```

Comment:
For consistency, can we add a space at the beginning and end of code within the curly braces?